### PR TITLE
Removed observability code.

### PR DESCRIPTION
**Pull request by mwhittaker**
A Service Weaver application has logs, traces, and metrics. Before this PR, `weaver kube deploy` generated Kubernetes YAML to set up Prometheus, Jaegar, Grafana, and Loki. The `weaver kube` babysitter exported logs, traces, and metrics to these systems. However, there was no way for a user to customize how telemetry was handled. If you wanted to export traces to Datadog, for example, well, you couldn't.

To address this problem in the long term, we're considering introducing a general purpose mechanism that would allow developers to customize deployers using a set of components. A metric exporting component, for example, would customize how metrics are exported.

In the short term, we're going to introduce a much simpler and less general approach where a developer can plug in custom ways to manage telemetry. This also means that `weaver kube` no longer has to generate YAML to run Prometheus, Jaegar, and whatnot. Instead, they will be plugins.

### CODE REVIEW

File: internal/impl/kube.go [hunk 1]
```diff
@@ -37,16 +39,21 @@ import (
...
 	appContainerName = "serviceweaver"
+
 	// The exported port by the Service Weaver services.
 	servicePort = 80
 
 	// Port used by the weavelets to listen for internal traffic.
 	internalPort = 10000
+
+	// Default Prometheus port.
+	prometheusPort = 9090
```
Comment (rgrandl) [inline internal/impl/kube.go:56]: is this also going away? If I remember correctly, we discussed to export only logs (to stdout) by default.

**Review by rgrandl — APPROVED** [review]
Looks great.

File: internal/impl/kube.go (hunk 1 above)
Comment (mwhittaker) [reply internal/impl/kube.go:56]: Yeah, I'll remove this in a future PR!

**Review by mwhittaker — COMMENTED** [review]
Thanks Robert!

**Pull request merged**